barnsbury/sass/_extra-child-theme.scss

```diff
@@ -393,3 +393,6 @@ a {
 		margin-top: $spacing_unit;
 	}
 }
+// Updates the Mobile Navigation to be next to the site title.
+$mobile-nav-side-right-margin: 100px;
+@import "../../varia/sass/components/header/_site-mobile-nav-side";
```

Comment:
This approach also relies on the parent theme being updated.

Do we need to import the CSS here? Can't we just add the CSS itself to the parent theme and then add one rule to change the right margin for each child?

Reply:
The imported css here gets expanded and and combined into the css that ships with the child themes. 
So it does rely on the parent theme but only during the development. So I think it is fine in this case.

Reply:
Oh good point :)